x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -25,6 +25,12 @@ interface IRequestParams<ResponseBody> {
   params?: object;
   hasValidData?: (body?: ResponseBody) => boolean;
 }
+interface IErrorResponse {
+  message: string;
+  attributes: {
+    errors: string[];
+  };
```

Comment:
FYI: The `attributes` key comes from [Kibana's response factory](https://github.com/elastic/kibana/blob/master/docs/development/core/server/kibana-plugin-core-server.kibanaresponsefactory.md) and as far as I could tell was the only way to pass custom data/objs through the response body. I tried setting other keys like `data` or just `errors` but they didn't come through on the front end, so I'm pretty sure only attributes works.